providers/efa/verbs.c

```diff
@@ -22,6 +22,9 @@
 #include "efadv.h"
 #include "verbs.h"
 
+#define field_avail(ptr, fld, sz) (offsetof(typeof(*ptr), fld) + \
```

Comment:
Silly question, why vext_field_avail() is not good?

Reply:
Thanks! wasn't aware of this function.
Its implementation is a bit different but it's a fair assumption.

Reply:
Can you send follow up patch to switch and use that vext_field_avail() function? So i'll be able to merge this PR

Reply:
and save you the delay

Reply:
Sure, will send it soon.
Thanks Leon